src/thunks/postFavorites.js

```diff
@@ -21,7 +21,7 @@ export const postFavorites = (movie, userId) => {
       }
       const result = await response.json();
       console.log(result)
-      // dispatch(addFavorite(result.message));
+      dispatch(addMessage(result.message));
```

Comment:
just about to test this file, so i'll merge this now and make sure the dispatches are tested. Thanks!